lib/matplotlib/axes/_subplots.py

```diff
@@ -186,6 +186,9 @@ def subplot_class_factory(axes_class=None):
     not have to be created for every type of Axes.
     """
     if axes_class is None:
+        cbook.warn_deprecated(
+            "3.3", message="Support for passing None to subplot_class_factory "
```

Comment:
Same thing about the alignment here as the other PR.

Reply:
at least for deprecations this pattern is common through the codebase, as can be checked with `git grep -A2 'message='`)